apps/web/src/components/templates/notification-setting-form/NotificationSettingsForm.tsx

```diff
@@ -124,6 +124,7 @@ export const NotificationSettingsForm = ({
                   error={fieldState.error?.message}
                   label="Notification Identifier"
                   description="This will be used to identify the notification template using the API."
+                  disabled={readonly}
```

Comment:
Is it still possible to click the copy button when disabled?

Reply:
yes, copy button works